src/triage/experiments/rq.py

```diff
@@ -0,0 +1,159 @@
+import logging
+import time
+from triage.component.catwalk.utils import Batch
+from triage.experiments import ExperimentBase
+try:
+    from rq import Queue
+except ImportError:
+    print('rq not available')
```

Comment:
Might want to beef this is up slightly? If I import this module I may or may not see the printed message; but, as is, this'll still allow me to instantiate `RQExperiment`, (and raise a `NameError` or something).

Perhaps you want to then `raise` (so as not to squash the `ImportError`)?